docs/regmap/adi_regmap_spi_engine.txt

```diff
@@ -1,6 +1,6 @@
 TITLE
 SPI Engine (axi_spi_engine)
-axi_spi_engine
+AXI_SPI_ENGINE
```

Comment:
Need to update also [axi-spi-engine.rst](https://github.com/analogdevicesinc/hdl/blob/9de7990027d698a892ec009bc13a58342a21cc39/docs/library/spi_engine/axi_spi_engine.rst?plain=1#L79) since this field is unique and case sensitive.
Change from
```
.. hdl-regmap::
   :name: axi_spi_engine
```
to
```
.. hdl-regmap::
   :name: AXI_SPI_ENGINE
```

Reply:
Done this change. Thank you.